Plugin/Snippets/Movement/AutoRotate3DFixedUpdate.cs

```diff
@@ -0,0 +1,14 @@
+﻿using UnityEngine;
+
+namespace LUT.Snippets
+{
+    public class AutoRotate3DFixedUpdate : MonoBehaviour
```

Comment:
This is already implemented!

https://github.com/Lawendt/LawendtUnityUtilities/blob/master/Plugin/Snippets/Movement/AutoRotate3D.cs

Reply:
It showed up an exception that there were two classes with the same name, so I changed the name of this one.